src/App.js

```diff
@@ -1,5 +1,41 @@
+import {
+  isCarNameValid,
+  isPlayerValid,
+  isTurnValid,
+} from "./utils/validation.js";
+import Race from "./Race.js";
+import { raceWinnerOutput } from "./output.js";
+import { findWinner } from "./utils/calculator.js";
+import { handleError } from "./utils/error.js";
+import { input } from "./input.js";
+
 class App {
-  async run() {}
+  async run() {
+    try {
+      let raceResult = {};
+
+      const carList = await input(
+        "경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)\n"
+      );
+      const cars = await carList.split(",").map((car) => car.trim());
```

Comment:
`carList`는 `input`을 통해 받아온 값입니다. 따라서, 해당 값은 `carList`라는 이름보다는 `carListInput`에 가까운 듯 합니다.
더 크게 본다면, 해당 로직은 다음과 같이 나타내는것이 더 적절할 듯 해요

```suggestion
      const carList = await input(
        "경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)\n"
      ).split(",").map((car) => car.trim());
```